src/PatientTestsApi/LoadPatient/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
```

Comment:
@daniel-rocha @hannesne I think it's better to use "authLevel": "function" instead. That's the reason why APIM needs to retrieve the host key in key vault. We may need to change it in other function.json as well.